### DISCUSSION TITLE
Sync tasks from remote calendars

### DISCUSSION THREAD

**Discussion by brandonpollack23** [davish/obsidian-full-calendar#432] [Feature Requests]
This is already amazing, and you seem to already have more work cut out, but if I could also download my "tasks" from nextcloud and access them from within Obsidian it'd be amazing.

Use case example:
Tasks are synced accross all devices and most of the time I just need to see a TODO item, but some TODO items have [markdown links](markdown.md) in them like this.  They just link to another file in my obsidian folder for more information.

I guess I could learn a bunch and make my own plugin, but this seems to already do most of the work.  I wonder how hard it would be to just add them or integrate with obsidian tasks?

**Comment by paulsybrandy** [comment]
I most definitely second this request! Thanks.
